lib/gui/app.js

```diff
@@ -297,10 +292,9 @@ app.controller('HeaderController', function(OSOpenExternalService) {
     const supportUrl = selectionState.getImageSupportUrl() || DEFAULT_SUPPORT_URL;
     OSOpenExternalService.open(supportUrl);
   };
-
 });
 
-app.controller('StateController', function($rootScope, $scope) {
+app.controller('StateController', function ($rootScope, $scope) {
```

Comment:
Angular relies on being able to set custom contexts with `.call()` and `.apply()`, which fat arrow functions don't support. Basically, Angular.js 2 is not ES6 ready.